### DISCUSSION TITLE
How to add a custom resolver?

### DISCUSSION THREAD

**Discussion by ncunningham-stairsupplies** [MichalLytek/typegraphql-prisma#121] [Q&A] [answered]
Im going through the process of adding a customer resolver to the generated resolvers. Im stuck at adding the type for the context. Im not sure where that would be located?

**Comment by MichalLytek** [comment, marked as answer]
Type for the context can be located in any file you want. You just need to import it when you want to create context object in a type-safe way. Check the docs of Apollo Server or other lib you use for exposing GraphQL schema to the world.

**Reply by ncunningham-stairsupplies** [reply]
okay that worked. my other question is if there is a way to get reference to the fastify instance from within the resolver?

**Reply by MichalLytek** [reply]
Yes, search in fastify-gql/mercurius docs.
